.github/workflows/cpp-lint.yml

```diff
@@ -9,6 +9,10 @@ on:
 jobs:
   cpp-linter:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        compiler: [clang++-14]
```

Comment:
it seems this matrix/environment isn't being used? the compiler is always hardcoded to 14 elsewhere?